src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -99,17 +93,42 @@ public void testDoGet_InvalidUser() throws IOException, ServletException {
   }
 
   @Test
-  public void testDoGet_isAdmin() throws IOException, ServletException {
-    Mockito.when(mockRequest.getSession().getAttribute("user")).thenReturn("test_username");
-
-    int fakenumOfUsers = mockUserStore.getAllUsers().size();
-    int fakenumOfConvos = mockConversationStore.getAllConversations().size();
-    int fakenumOfMessages = mockMessageStore.getAllMessages().size();
+  public void testDoGet_AdminStatsPage() throws IOException, ServletException {
+    List<User> fakeUserList = new ArrayList<>();
+    User fakeUser = new User(UUID.randomUUID(), "test_username", "fakePasswordHash", Instant.now());
+    fakeUser.setAdmin(true);
+    fakeUserList.add(fakeUser);
+    Mockito.when(mockUserStore.getUser("test_username")).thenReturn(fakeUser);
+    List<String> fakeAdminList = new ArrayList<String>();
```

Comment:
Consistency: Lines 97 vs. 102 (and in other areas)
`ArrayList<>` vs. `ArrayList<String>`